docs/comparisons.md

```diff
@@ -111,3 +111,19 @@ virtualenvs with their dependencies installed.
 - fades can only run individual script files while pipx can also run packages.
 
 Example interaction: None.
+
+## pipx vs pae/pactivate
+
+_pae_ is a Bash command-line function distributed with [pactivate](https://github.com/cynic-net/pactivate) that uses pactivate to create non-ephermeral environments that can be used for both command-line applications and libraries.
+
+- pae and pipx both create isolated environments without having to specify (and remember) a directory in which to store them.
+- pae allows you to use any Python interpreter available on your system, so you can use arbitrary versions of Python.
+- pae does not require you to have anything other than a Python interpreter available; it downloads and bootstraps `pip` and `virtualenv`, always using the latest releases of both.
```

Comment:
Same for pipx, assuming your using the zipapp packaging.

Reply:
Actually, it's not quite the same. `pipx` can still fail in more cases on OS-supplied "minimal" Python interpreter installations. For example, on my Debian 12 system, `python3 ~/Downloads/pipx.pyz  install grip` fails with:

```
The virtual environment was not created successfully because ensurepip is not
available.  On Debian/Ubuntu systems, you need to install the python3-venv
package using the following command.

    apt install python3.11-venv

You may need to use sudo with that command.  After installing the python3-venv
package, recreate your virtual environment.
```

(pae downloads and installs the latest `virtualenv` package, and so does not rely on the Python installation providing that, as mine doesn't above. I'd not realised that you're using the venv subset of virtualenv instead; I've added a note about that to the comparison.)

That's not to say that pae is perfect in this regard; it runs into a similar issue on Debian 10 and below, where I print a similar message when the OS is lacking `python3-distutils`.